go/cmd/vtctldclient/internal/command/vschemas.go

```diff
@@ -141,7 +148,29 @@ func commandGetVSchema(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+var validateVSchemaOptions = struct {
+	Keyspace      string
```

Comment:
nope :), your usage has this as a positional arg (which is correct), and you get this via `cmd.Flags().Arg(0)`